test/test_fields.py

```
@@ -26,18 +45,55 @@ def test_model_json_schema(self):
+        Order(state=State.PREPARATION)  # Should not raise
+
+    def test_enum_field_dump_json(self):
+        order = Order(state=State.PREPARATION)
+        dump = order.model_dump_json()
+        assert '{"state":"Preparation"}' == dump
+
+    def test_workarround_enum(self):
+        workarround = WorkarroundModel(state=WorkarroundEnum.OK)
+        assert workarround.state == WorkarroundEnum.OK
+        dump = workarround.model_dump_json()
+        assert '{"state":"Ok"}' == dump
```

Ruff (0.8.2)

Yoda condition detected

Rewrite as `dump == '{"state":"Ok"}'`

(SIM300)